test/http_ex/backend/mock_test.exs

```diff
@@ -183,5 +183,124 @@ defmodule HTTPEx.Backend.MockTest do
                      Mock.verify!(self())
                    end
     end
+
+    test "min_calls not met" do
+      Mock.expect_request!(
+        endpoint: "http://www.example.com",
+        expect_body: "GET",
```

Comment:
If you remove this `expect_body`, you don't have to pass in the body in the requests.